pyproject.toml

```diff
@@ -35,12 +35,11 @@ version = "0.8.5"
 
 dependencies = [
     "aiohttp == 3.9.4",
-    "aiobotocore == 2.5.0",
+    "aiobotocore == 2.13.0",
```

Comment:
`requirements.txt` should be updated as well

Reply:
done